src/microbe_stage/SpawnSystem.cs

```diff
@@ -303,10 +303,10 @@ private void DespawnEntities(Vector3 playerPosition)
             }
 
             var entityPosition = ((Spatial)entity).Translation;
-            var distance = (playerPosition - entityPosition).Length();
+            var squaredDistance = (playerPosition - entityPosition).LengthSquared();
 
             // If the entity is too far away from the player, despawn it.
-            if (distance > spawned.DespawnRadius)
+            if (squaredDistance > spawned.DespawnRadius * spawned.DespawnRadius)
```

Comment:
In the old code `spawned.DespawnRadius` was already the squared distance so, that avoided also this one multiplication.